src/main/index.js

```diff
@@ -97,7 +97,7 @@ if (process.platform === "win32") {
       return spawn("cmd.exe", ["/c", "mkdir", path.join(USERDATA_PATH, "extras")])
     })
     .then(()=> {
-      return spawn("cmd.exe", ["/c", "mkdir", path.join(USERDATA_PATH, "workspaces")])
+      return spawn("cmd.exe", ["/c", "mkdir", path.join(USERDATA_PATH, "ui/workspaces")])
     })
     .then(()=> {
       return spawn("cmd.exe", ["/c", "mkdir", path.join(USERDATA_PATH, "default")])
```

Comment:
This magic "spawn" logic is here to ensure that real directories are created (required to be able to execute "extras" [such as postgres] in correct context). It may be removed in a subsequent change.